src/relay/backend/contrib/ethosn/codegen_ethosn.h

```diff
@@ -227,7 +227,9 @@ NetworkWithIDs ConstructNetwork(const IRModule& mod, const GlobalVar& var, const
 /*! \brief Attributes to store the compiler options for Ethos-N */
 struct EthosnCompilerConfigNode : public tvm::AttrsNode<EthosnCompilerConfigNode> {
   String variant;
-  int sram_size_bytes;
+  String sram_size;
```

Comment:
Why can't these be numeric types?

Reply:
Using String type simplifies passing the value to the driver stack.